File: Python/Arrays/removeDuplicatesFromSortedArray.py
def removeDuplicates(arr):
    res = 1
    for i in range(1, len(arr)):
        if(arr[i-1]!=arr[i]):
            arr[res] = arr[i]
            res = res + 1
    return res

if __name__ == "__main__":
    n = int(input("Enter the size of the array : "))
    arr = list()
    print("Enter the elements of the array : ")
    for i in range(0, n):
        k = int(input())
        arr.append(k)
    n = removeDuplicates(arr)
    print("Array after rmoving the duplicates from the sorted array : ")
    for i in range(n):
        print(arr[i])

File: Java/Arrays/removeDuplicatesFromSortedArray.java
import java.util.*;

class removeDuplicatesFromSortedArray{

    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the size of the array : ");
        int n = sc.nextInt();
        ArrayList<Integer> arr = new ArrayList<>();
        System.out.println("Enter the elements of the array : ");
        for(int i=0;i<n;i++){
            int k = sc.nextInt();
            arr.add(k);
        }
        n = removeDuplicates(arr);
        System.out.print("Elements after removing the duplicates : ");
        for(int i=0;i<n;i++){
            System.out.print(arr.get(i) + " ");
        }
    }

    public static int removeDuplicates(ArrayList<Integer> arr){
        int res = 1;
        for(int i=1;i<arr.size();i++){
            if(arr.get(i-1) != arr.get(i)){
                arr.set(res, arr.get(i));
                res = res + 1;
            }
        }
        return res;
    }
};